localstack/services/transcribe/provider.py

```diff
@@ -136,7 +139,16 @@ def start_transcription_job(
         if not output_key:
             output_key = f"{job_name}.json"
 
-        transcript = Transcript(TranscriptFileUri=f"s3://{output_bucket}/{output_key}")
+        s3_client = aws_stack.connect_to_service("s3")
+
+        # the presign url is valid for 15 minutes
+        presign_url = s3_client.generate_presigned_url(
+            "put_object",
+            Params={"Bucket": output_bucket, "Key": output_key},
+            ExpiresIn=60 * 15,
+        )
```

Comment:
question: what is this presigned URL for? Is it for the client to then retrieve the result of the job? should it be `get_object` then?
Also, the related issue says it's when calling `GetTranscriptionJob`, is the pre-signed URL generated every time we call `GetTranscriptionJob`? Because now it would be valid for 15min when starting the job, and when calling `GetTranscriptionJob` 20min after the beginning of the job, the URL would be expired. Is that how it works?

Reply:
>  should it be `get_object` then?

Yes, the change was already in pipeline and hence the draft 😃, but thanks for pointing it out 🙏. 


> Also, the related issue says it's when calling GetTranscriptionJob, is the pre-signed URL generated every time we call GetTranscriptionJob? Because now it would be valid for 15min when starting the job, and when calling GetTranscriptionJob 20min after the beginning of the job, the URL would be expired. Is that how it works?

Yes @bentsku, its mentioned in one of the NOTE of the [AWS transcribe service docs](https://docs.aws.amazon.com/cli/latest/reference/transcribe/start-transcription-job.html).

Reply:
> Note
> Temporary URIs for service-managed Amazon S3 buckets are only valid for 15 minutes. If you get an AccesDenied > > error, you can get a new temporary URI by running a GetTranscriptionJob or ListTranscriptionJob request.

So it is what I was saying, the URL is generated when calling `GetTranscriptionJob` and not when starting the job I think. I think that's a big logical change and we should do it too in `GetTranscriptionJob`.
Also, we're returning the `Transcript` field from `StartTranscriptionJob` but it's not supposed to be there, right?